src/common/cockpitpipetransport.c

```diff
@@ -117,6 +117,8 @@ on_pipe_close (CockpitPipe *pipe,
             problem = "terminated";
           else if (is_cockpit && WIFEXITED (status) && WEXITSTATUS (status) == 127)
             problem = "no-cockpit";      // cockpit-bridge not installed
+          else if (is_cockpit && WIFEXITED (status) && WEXITSTATUS (status) == 1)
+            problem = "access-denied";   // disabled user shell
```

Comment:
Lets expand this comment a bit better. "A disabled eg: /bin/false or /sbin/nologin will return status of 1"

Reply:
Agreed, PR updated.